stackslib/src/net/download/nakamoto/tenure_downloader_set.rs

```diff
@@ -67,6 +67,35 @@ use crate::net::server::HttpPeer;
 use crate::net::{Error as NetError, Neighbor, NeighborAddress, NeighborKey};
 use crate::util_lib::db::{DBConn, Error as DBError};
 
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub(crate) struct CompletedTenure {
+    tenure_id: ConsensusHash,
+    start_block: StacksBlockId,
+    end_block: StacksBlockId,
+}
+
+impl From<&TenureStartEnd> for CompletedTenure {
+    fn from(tse: &TenureStartEnd) -> Self {
+        Self {
+            tenure_id: tse.tenure_id_consensus_hash.clone(),
+            start_block: tse.start_block_id.clone(),
+            end_block: tse.end_block_id.clone(),
+        }
+    }
+}
+
+impl From<&mut NakamotoTenureDownloader> for CompletedTenure {
+    fn from(ntd: &mut NakamotoTenureDownloader) -> Self {
+        Self {
+            tenure_id: ntd.tenure_id_consensus_hash,
+            start_block: ntd.tenure_start_block_id,
+            end_block: ntd.tenure_end_block_id,
+        }
+    }
+}
+
+pub const PEER_DEPRIORITIZATION_TIME: u64 = 60;
```

Comment:
nit: I assume this is in seconds? would be good to add the unit to the var name.

Reply:
Yup, will update